tests/test_gousto.py

```diff
@@ -25,27 +25,28 @@ def test_yields(self):
         self.assertEqual("2 serving(s)", self.harvester_class.yields())
 
     def test_ingredients(self):
+        print("ingredients list", self.harvester_class.ingredients())
         self.assertCountEqual(
             [
                 "1 onion",
                 "150g chestnut mushrooms",
                 "1 garlic clove",
                 "10g fresh parsley",
-                "30g rennet-free parmesan †",
+                "30g rennet-free parmesan",
                 "2 British pork loin steaks",
-                "1 pot of double cream (227ml) †",
-                "1/2 beef stock cube †",
-                "200g linguine †",
-                "Olive oil",
-                "pepper",
-                "salt",
+                "1 pot of double cream (227ml)",
+                "1/2 beef stock cube",
+                "200g linguine"
+                # "Olive oil",
```

Comment:
Is there a reason that these are commented out as opposed to being removed entirely?